indexer/services/ender/__tests__/handlers/perpetual-market-handler.test.ts

```
@@ -89,12 +92,20 @@ describe('perpetualMarketHandler', () => {
+    [
+      'PerpetualMarketCreateEventV3',
+      3,
+      PerpetualMarketCreateEventV3.encode(defaultPerpetualMarketCreateEventV3).finish(),
+      expectPerpetualMarketV3,
+      defaultPerpetualMarketCreateEventV3,
+    ],
   ])('%s', (
     _name: string,
     version: number,
     perpetualMarketCreateEventBytes: Uint8Array,
     expectPerpetualMarket: Function,
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)